sky/check.py

```diff
@@ -245,10 +245,10 @@ def _format_enabled_cloud(cloud_name: str) -> str:
                 # here we are using rich. We should migrate this file to
                 # use colorama as we do in the rest of the codebase.
                 symbol = ('└── ' if i == len(existing_contexts) - 1 else '├── ')
-                contexts_formatted.append(f'\n        {symbol}{context}')
+                contexts_formatted.append(f'\n    {symbol}{context}')
```

Comment:
Reduce the indent for cleaner view, cc'ing @romilbhardwaj : )

Reply:
hmm I prefer the older format with the indent since it visually distinguishes the context as a sub-item under Kubernetes, making it clearer that it's part of Kubernetes rather than another top-level entry (cloud).

<img width="285" alt="image" src="https://github.com/user-attachments/assets/b1ff6aa1-4453-4b52-a545-56f66ae639c6" />
<img width="289" alt="image" src="https://github.com/user-attachments/assets/c8865325-917c-47da-b6e6-202da9fc73d7" />

Not feeling too strongly though

Reply:
I feel the color should distinguish it well. The problem I had with the indent was that when my contexts are long and multiple contexts are specified in `~/.sky/config.yaml`. The multiple level of indents make it a bit annoying:

<img width="421" alt="image" src="https://github.com/user-attachments/assets/3bc4444b-2459-476e-893f-1c2b24d8d5a5" />